Lesson_11/test_logger.py

```diff
@@ -0,0 +1,39 @@
+# test_logger.py
+import unittest
+from unittest.mock import patch
+import logging
+from logger import log_event
+
+
+class TestLogEvent(unittest.TestCase):
+
+    @patch('logger.logging.getLogger')
```

Comment:
It is no need to mock logger as it has real implementation. You can only read file with logs messages after function called, read them and compare with expected result.